### PR TITLE
Sup 12502 google structured data

### PR DESCRIPTION
**Pull request by ErezBouskila**

### CODE REVIEW

**Comment by ErezBouskila** [comment]
@OrenMe Please review PR.
Thanks

File: lib/EmbedCodeGenerator.js [hunk 1]
```diff
@@ -457,7 +457,10 @@ EmbedCodeGenerator.prototype = {
 			data['embedMethod'] = (params.embedType == 'dynamic') ? 'embed' : 'thumbEmbed';
 			data['kWidgetObject'] = this.getEmbedObject( params );
 		}
-
+        var date = parseInt(params.entryMeta.uploadDate);
```
Comment (OrenMe) [inline lib/EmbedCodeGenerator.js]: Are you sure this nested oath will exist?

File: lib/EmbedCodeGenerator.js [hunk 2]
```diff
@@ -457,7 +457,10 @@ EmbedCodeGenerator.prototype = {
...
+        if (date.toString().length === 10) {
```
Comment (OrenMe) [inline lib/EmbedCodeGenerator.js]: not clear what this check is? also, why do you move from sting to int the line above just to parse to string back here?

File: lib/templates.js [hunk 3]
```diff
@@ -294,8 +296,10 @@ function program1(depth0,data) {
     + escapeExpression(((stack1 = ((stack1 = depth0.entryMeta),stack1 == null || stack1 === false ? stack1 : stack1.description)),typeof stack1 === functionType ? stack1.apply(depth0) : stack1))
     + "\"></span>\n<span itemprop=\"duration\" content=\""
     + escapeExpression(((stack1 = ((stack1 = depth0.entryMeta),stack1 == null || stack1 === false ? stack1 : stack1.duration)),typeof stack1 === functionType ? stack1.apply(depth0) : stack1))
-    + "\"></span>\n<span itemprop=\"thumbnail\" content=\""
+    + "\"></span>\n<span itemprop=\"thumbnailUrl\" content=\""
```
Comment (OrenMe) [inline lib/templates.js]: is this an existing bug? when did this change?

File: dist/KalturaEmbedCodeGenerator.js [hunk 4]
```diff
@@ -1,6 +1,6 @@
-/*! Kaltura Embed Code Generator - v1.0.7 - 2013-03-16
+/*! Kaltura Embed Code Generator - v1.0.7 - 2018-02-13
```
Comment (OrenMe) [inline dist/KalturaEmbedCodeGenerator.js]: Need to update the version so the build will be 1.1.0

**Pull request merged**